src/core/fetchers/types.ts

```diff
@@ -29,3 +31,7 @@ export type RequestCallback = (
 export type ErrorType<TErrorData> = TErrorData;
 
 export type BodyType<TBodyData> = TBodyData;
+
+type QueryOperations = keyof typeof operationNames.Query;
+type MutationOperations = keyof typeof operationNames.Mutation;
+export type Operations = QueryOperations | MutationOperations;
```

Comment:
Please consider your placement of these types.

This is a generic file with types for fetchers used across multiple services. Now you have added a new type which is only relevant to a specific service. You also use very specific names `QueryOperations`, `MutationOperations` and `Operations`.

I would prefer is this was kept within the `dbc-gateway` (which is the service in question) or Cypress tests (where the types are used).

If there is a larger plan with this - e.g. that this should include all operations for all GraphQL services then I would expect it to be mentioned in a code comment or the like.